lib/mainloop-control.c

```diff
@@ -450,6 +452,68 @@ export_config_graph(ControlConnection *cc, GString *command, gpointer user_data,
   control_connection_send_reply(cc, result);
 }
 
+static void
+control_internal_logs(ControlConnection *cc, GString *command, gpointer user_data, gboolean *cancelled)
+{
+  GString *result = g_string_new("");
+  gchar **arguments = g_strsplit(command->str, " ", 0);
+  static AFInterLive ret = AFINTER_LIVE_COLLECTION_NONE;
```

Comment:
Please do not try to double-manage the internal state of the live collection.

both `afinter_stop_live_collection` and `afinter_get_size_of_internal_logs` can give you the state back, like in the case of `afinter_start_live_collection` you did

or, alternatively, you can add a public state-getter if it is really needed (but, I guess this is not that case, as I mentioned above, both functions can signal the state back like `afinter_start_live_collection` does

Reply:
@HofiOne : okay, done.